engine/dlib/src/test/wscript

```diff
@@ -122,8 +130,8 @@ def build(bld):
     create_test(bld, 'test_spinlock', extra_libs = ['THREAD'])
     create_test(bld, 'test_sys', extra_libs = ['THREAD'], extra_defines = extra_defines)
     create_test(bld, 'test_template', extra_libs = ['THREAD'])
-    create_test(bld, 'test_ssdp_internals', extra_libs = ['THREAD'])
-    create_test(bld, 'test_ssdp', extra_libs = ['THREAD'], skip_run = skip_run)
+    create_test(bld, 'test_ssdp_internals', extra_libs = ['THREAD'], skip_run = skip_run or skip_ssdp)
+    create_test(bld, 'test_ssdp', extra_libs = ['THREAD'], skip_run = skip_run or skip_ssdp)
```

Comment:
Not sure why certain tests doesn't work under WSL (possibly because it's under Windows)